src/analysis/classic/hunter/shared/KillShot.tsx

```diff
@@ -0,0 +1,77 @@
+import SPELLS from 'common/SPELLS/classic/hunter';
+import { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import Abilities from 'parser/core/modules/Abilities';
+import SPELL_CATEGORY from 'parser/core/SPELL_CATEGORY';
+import ExecuteHelper from 'parser/shared/modules/helpers/ExecuteHelper';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import ItemDamageDone from 'parser/ui/ItemDamageDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+
+import { KILL_SHOT_EXECUTE_RANGE } from './constants';
+
+class KillShot extends ExecuteHelper {
+  static executeSources = SELECTED_PLAYER;
+  static lowerThreshold = KILL_SHOT_EXECUTE_RANGE;
+  //static singleExecuteEnablers: Spell[] = [TALENTS.HUNTERS_PREY_TALENT, SPELLS.DEATHBLOW_BUFF];
```

Comment:
Remove comment

Reply:
I will have to grep for comments since clearly I am not good and removing them when debugging/cleaning up.
done